website/src/middlewares/api-v3/developmentMode.js

```diff
@@ -0,0 +1,12 @@
+import nconf from 'nconf';
```

Comment:
This file should have a unit test. Should be able to just stub the result of `nconf.get` for it.